beta.dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG NODE_VERSION=18
+ARG JDK_VERSION=23
+
+################################################################################
+# Stage 1: Create a stage for building frontend client.
```

Comment:
Maybe we should adjust this stage description so that it fits with others

Reply:
Check out if this is what you mean.